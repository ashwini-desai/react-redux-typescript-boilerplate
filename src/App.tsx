import { Text } from '@medly-components/core';
import React from 'react';
import { hot } from 'react-hot-loader/root';

export const App: React.SFC = () => (
    <Text textSize="L3" textWeight="Strong">
        React Redux Typescript boilerplate
    </Text>
);

export default hot(App);
