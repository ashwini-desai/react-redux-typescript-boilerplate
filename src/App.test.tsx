import { render } from '@test-utils';
import React from 'react';
import { App } from './App';

describe('App', () => {
    it('should render properly', () => {
        const { container } = render(<App />);
        expect(container).toMatchSnapshot();
    });
});
