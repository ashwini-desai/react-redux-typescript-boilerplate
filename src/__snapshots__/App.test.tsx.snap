// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`App should render properly 1`] = `
.c0 {
  font-size: 40px;
  font-weight: 700;
  color: inherit;
}

<div>
  <strong
    class="c0"
  >
    React Redux Typescript boilerplate
  </strong>
</div>
`;
